Web Quiz Engine (Kotlin)/task/src/engine/entity/AnswerQuizRequestBody.kt

```diff
@@ -0,0 +1,3 @@
+package engine.entity
+
+data class AnswerQuizRequestBody(val answerList: List<Int>?)
```

Comment:
Typically, the entity package is reserved for database entities.
For Resquest and Response body objects I would put them in a `model` package.

Reply:
That makes sense. I don`t like `model` but I will -begrudgingy- use it 😜

Reply:
Feel free to name the package anything other than `model`.
The idea is to have a separation between your database entities and your REST POJOs.